tools/samba/illumina_suite/qiime2_dbotu_q2.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!--
+Copyright (c) 2024, QIIME 2 development team.
+
+Distributed under the terms of the Modified BSD License. (SPDX: BSD-3-Clause)
+-->
+<!--
+This tool was automatically generated by:
+    q2galaxy (version: 2024.10.0)
+for:
+    qiime2 (version: 2024.10.1)
+-->
+<tool name="qiime2 feature-table summarize" id="qiime2__feature_table__summarize" version="2024.10.0+q2galaxy.2024.10.0" profile="22.05" license="BSD-3-Clause">
+    <description>Summarize table</description>
+    <xrefs>
+        <xref type="bio.tools">qiime2</xref>
+    </xrefs>
+    <requirements>
+        <container type="docker">quay.io/qiime2/amplicon:2024.10</container>
+    </requirements>
+    <version_command>q2galaxy version dbotu_q2</version_command>
+    <command detect_errors="exit_code">q2galaxy run dbotu_q2 call_otus '$inputs'</command>
+    <configfiles>
+        <inputs name="inputs" data_style="staging_path_and_source_path"/>
+    </configfiles>
+    <inputs>
+        <param name="table" type="data" format="qza" label="table" help=""/>
```

Comment:
Same issue I don't know the inputs and the format 
same pb for the outputs I just guessed